wren-ai-service/src/providers/llm/openai.py

```diff
@@ -90,7 +90,8 @@ async def run(
         logger.debug(f"Running AsyncOpenAI generator with prompt: {prompt}")
         message = ChatMessage.from_user(prompt)
         if self.system_prompt:
-            messages = [ChatMessage.from_system(self.system_prompt), message]
+            # updated from_system to from_assistent as the new openai api is not accepting system prompts anymore, only user and assistant.
```

Comment:
thanks for the info, could you also provide relevant release note or sth I can look further?

Reply:
https://community.openai.com/t/o1-models-do-not-support-system-role-in-chat-completion/953880/2 
This is for o1 models and they would change it for all.